src/App.vue

```diff
@@ -27,21 +40,34 @@ export default defineComponent({
   <header>
     <NavBar />
   </header>
-  <main>
-    <router-view class="main-view"></router-view>
-  </main>
+  <template v-if="error">
+    <div class="error">
+      <p>{{ error }}</p>
+      <br />
+      <p>{{ errorInstance }}</p>
+      <br />
+      <p>{{ errorInfo }}</p>
```

Comment:
Error handling is mostly for users. Do you think it gives your **users** some value to see `errorInstance`& `errorInfo`? 🤔 

Displaying `error` may have some value, but also it may be just some random JS error that does not have any sense for non-dev users. Maybe it's worth to add also some human-readable message? 😁

Reply:
That was a bit selfish of me, but it's fixed now - something more user friendly :+1:

Reply:
💚